addons/l10n_ar/views/account_move_view.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_account_move_filter" model="ir.ui.view">
+        <field name="name">account.move.filter</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_account_move_filter"/>
+        <field name="arch" type="xml">
+            <field name="partner_id" position="after">
+                <field name="l10n_ar_afip_responsibility_type_id"/>
+            </field>
+            <group>
+                <filter string="AFIP Responsibility Type" domain="[]" name="l10n_ar_afip_responsibility_type_id_filter" context="{'group_by':'l10n_ar_afip_responsibility_type_id'}"/>
+            </group>
+        </field>
+    </record>
+
+
+    <record id="view_move_form" model="ir.ui.view">
+        <field name="name">account.move.form</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form"/>
+        <field name="arch" type="xml">
+            <field name="fiscal_position_id" position="before">
+                <field name="l10n_ar_afip_responsibility_type_id" options="{'no_open': True, 'no_create': True}" attrs="{'invisible': ['|', ('state', 'not in', ['open', 'paid']), ('l10n_latam_country_code', '!=', 'AR')]}"/>
+            </field>
+
+            <group id="header_left_group">
+                <field name='l10n_ar_afip_concept' attrs="{'invisible': ['|', ('l10n_latam_country_code', '!=', 'AR'), ('l10n_latam_use_documents', '=', False)]}"/>
```

Comment:
I am not sure about what you mean here. Aare you talking about remove the l10n_ar_afip_concept field from the view?

At the moment is a compute readonly field, We leave it visible because we think that let the user context in the view about the other fields that are shown and that depends on then (the service start/stop date field)